From 3b948aa33a01a09f6014e67389f267bd60a9c43e Mon Sep 17 00:00:00 2001
From: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
Date: Tue, 9 Nov 2021 15:25:44 +0200
Subject: [PATCH 3/5] HACK: Allow DomD enumerate PCI devices

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
---
 drivers/xen/pci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/xen/pci.c b/drivers/xen/pci.c
index 72d4e3f193af..1d8ec28bd260 100644
--- a/drivers/xen/pci.c
+++ b/drivers/xen/pci.c
@@ -206,8 +206,10 @@ static struct notifier_block device_nb = {
 
 static int __init register_xen_pci_notifier(void)
 {
+#if 0
 	if (!xen_initial_domain())
 		return 0;
+#endif
 
 	return bus_register_notifier(&pci_bus_type, &device_nb);
 }
-- 
2.43.0

